app/controllers/api/api_users_controller.rb

```diff
@@ -68,4 +23,10 @@ def valid_for(param)
     time_s = User::Api::VALID_FOR_OPTIONS.invert[time].to_s
     t('flashes.api.api-users.update.time.' + time_s)
   end
+
+  class << self
+    def model_class
```

Comment:
can't we have a class_attribute for declaring this? if the class attribute is set, it takes it ... otherwise it searches it by controller name ? https://apidock.com/rails/Class/class_attribute 
custom_model_class = ::User::Api

lot's of code for just declaring this, isn't it ?